src/Extractor/Hive.php

```diff
@@ -4,68 +4,83 @@
 
 namespace Keboola\DbExtractor\Extractor;
 
+use Keboola\DbExtractor\Adapter\ExportAdapter;
+use Keboola\DbExtractor\Adapter\Metadata\MetadataProvider;
+use Keboola\DbExtractor\Adapter\ODBC\OdbcConnection;
+use Keboola\DbExtractor\Adapter\ODBC\OdbcExportAdapter;
+use Keboola\DbExtractor\Adapter\ODBC\OdbcNativeMetadataProvider;
+use Keboola\DbExtractor\Adapter\Query\DefaultQueryFactory;
+use Keboola\DbExtractor\Adapter\ResultWriter\DefaultResultWriter;
+use Keboola\DbExtractor\Connection\HiveDnsFactory;
+use Keboola\DbExtractor\Connection\HiveOdbcConnection;
 use Keboola\DbExtractor\TableResultFormat\Exception\ColumnNotFoundException;
-use Dibi\Connection;
-use Keboola\DbExtractor\Connection\HiveConnectionFactory;
 use Keboola\Datatype\Definition\GenericStorage;
 use Keboola\DbExtractor\Exception\UserException;
+use Keboola\DbExtractor\TableResultFormat\Metadata\Manifest\DefaultManifestSerializer;
+use Keboola\DbExtractor\TableResultFormat\Metadata\Manifest\ManifestSerializer;
+use Keboola\DbExtractorConfig\Configuration\ValueObject\DatabaseConfig;
 use Keboola\DbExtractorConfig\Configuration\ValueObject\ExportConfig;
-use Psr\Log\LoggerInterface;
 
 class Hive extends BaseExtractor
 {
-    use DibiSupportExtractorTrait;
+    public const INCREMENTAL_TYPES = [
```

Comment:
Tu bol preklep ... nizsie v kode sa volalo `getBaseType` a nie `getType` ... fixol som to ... je to teda zoznam NATIVNYCH hive typov, nad ktorymi je mozne spravit incremental fetching -> je to pokryte datadir testami.